usr/share/rear/restore/TSM/default/400_restore_with_tsm.sh

```diff
@@ -4,15 +4,25 @@
 local num=0
 local filespace=""
 local dsmc_exit_code=0
-Print "File restore progression can be followed with tail -f \"$TMP_DIR/TSM-restore.log\""
+
+# Create common backup restore log file name prefix (same for each filespace):
+mkdir -p $VAR_DIR/restore
+local backup_restore_log_file=""
+local backup_restore_log_file_prefix=$BACKUP
+# Cf. how the contains_visible_char() function in lib/global-functions.sh is implemented:
+test "$CONFIG_APPEND_FILES" && backup_restore_log_file_prefix=$backup_restore_log_file_prefix-$( tr -d -c '[:graph:]' <<<"$CONFIG_APPEND_FILES" )
+
 for num in $TSM_RESTORE_FILESPACE_NUMS ; do
     filespace="${TSM_FILESPACES[$num]}"
+    LogUserOutput "Restoring TSM filespace $filespace"
+    # Create backup restore log file name (a different one for each filespace):
+    backup_restore_log_file=$VAR_DIR/restore/$backup_restore_log_file_prefix-$filespace-restore.log
```

Comment:
The issue here is that `$fileplace` are PATH like "/ /home/ /var/lib/". Then your variable looks like:
`backup_restore_log_file=TSM-/var/lib/-restore.log`. We have to substitute "/" by something like "_" to avoid issues.

I propose something like: 
```
backup_restore_log_file=$VAR_DIR/restore/$backup_restore_log_file_prefix-ROOT${filespace//\//_}-restore.log
```
This will generate `backup_restore_log_file=TSM-ROOT_var_lib_-restore.log`

Reply:
Many thanks for the information!
I will improve it to get a readable file name.